Documentation/Localization.ru_RU/Pages/AddingMultiple/Index.rst

```diff
@@ -0,0 +1,36 @@
+.. include:: /Includes.rst.txt
```

Comment:
This looks like new Translations and not just a bug to me? Or did you accidently forget to commit these pages?

Reply:
Fixed some bugs in the code, updated from local repository.
Everything seems to be in place, made new Pull requests

Reply:
There was a mix-up at commits

Reply:
I'll do another batch of translation a little later.

Reply:
Ok, so let us change the title of this pull request. As we are squashing the commits this will be the title of the final commit